xpano/gui/panels/sidebar.cc

```diff
@@ -213,6 +213,9 @@ Action DrawHelpMenu() {
     if (ImGui::MenuItem("About")) {
       action |= {ActionType::kShowAbout};
     }
+    if (ImGui::MenuItem("Report a bug")) {
```

Comment:
Please put the bug report above the About pane. It is usual to have the About pane as the last entry.